File: 프로그래머스/lv1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.js
function solution(n) {
  var answer = '';
  temp = '수박'
  if (n%2 === 0 ) { answer = temp.repeat(n/2)}
  else { answer = temp.repeat(parseInt(n/2))+'수'}
  return answer;
}

// 3 -> 수박수, 5 ->수박수박수 
// 4 -> 수박수박, 6 -> 수박수박수박
console.log(solution(4))

File: 프로그래머스/1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.py
def solution(n):
    return '수박'*(n//2) if (n%2) == 0 else '수박'*(n//2) + '수'
    # 레퍼런스 아래 방법이 오히려 더 깔끔한듯
    # str = "수박"*n
    # return str[:n]